test.lua

```diff
@@ -452,15 +493,26 @@ test:test("multiple expires test", function(test)
         }
     )
     -- test first expire part
-    fiber.sleep(1 + expire_delta)
-    local task = expirationd.task("test")
-    local cnt = task.expired_tuples_count
-    test:ok(cnt < tuples_count and cnt > 0, 'First part expires done')
+    local res = wait_cond(
+        function()
+            local task = expirationd.task("test")
+            local cnt = task.expired_tuples_count
```

Comment:
Shouldn't we compare this value with actual records count in space? (In similar way it does here - https://github.com/tarantool/expirationd/pull/81/files#diff-8cb3361cd200c46658a6738e92c421b0589c52742f6160315b03bc4192a2951fR527)